mobile/build.gradle

```diff
@@ -4,3 +4,27 @@
  * This is a general purpose Gradle build.
  * Learn how to create Gradle builds at https://guides.gradle.org/creating-new-gradle-builds
  */
+
+android {
```

Comment:
A problem occurred evaluating root project 'mobile'.
> Could not find method android() for arguments [build_8hzd5qy43o6j5reh449suhlwb$_run_closure1@65c69cd5] on root project 'mobile' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s


este error fica acontecendo ao tentar executar o gradle atraves do command ./gradlew assembleRelease indicado no tutorial

Reply:
Seguindo esse link https://www.instamobile.io/android-development/generate-react-native-release-build-android/
 claramente se ve que ele esta utilizando o android studio, simplesmente pela forma que esta estruturado os arquivos..
a questao é que quando se cria um projeto base no android studio, ele ja pre-configura certos dados nos arquivos de build e properties do gradle, oque no nosso caso nao existe..

Reply:
Eu estou seguindo este link https://developer.android.com/studio/build#module-level
como forma de orientação detalhada para configurar o necessario, mas como pode ver, nao deu certo

Reply:
Pois é. A dúvida era essa, como é um projeto expo, pode ser q não funcione.
Vou ver se encontro algo.

Reply:
Esse é o problema de depender de ferramentas tão específicas. Pois já temos o React Native e por cima dele vem o expo. 
Mas no site do react native tem um processo semelhante ao mostrado no outro link. Tenta aí: https://reactnative.dev/docs/signed-apk-android

Reply:
Pelo jeito, só fazendo um `expo eject` que remove a dependência do expo e volta o projeto para React Native puro. https://forums.expo.io/t/building-app-locally-without-ejecting/1874

Mas aí não dá

Reply:
Bem, a solução na [documentação oficial do expo está aqui](https://docs.expo.io/distribution/turtle-cli/?redirected). Agora é testar.

Reply:
Realmente esta ficando bem robusto toda essa estrutura do projeto...
Vou ler toda a documentação dos links e aplicar a solução